### PR TITLE
docs: releae notes 1.44 for js

### PR DESCRIPTION
**Pull request by yury-s**

### CODE REVIEW

File: docs/src/release-notes-js.md [hunk 1]
```diff
@@ -6,6 +6,143 @@ toc_max_heading_level: 2
...
+- New [`method: Page.removeLocatorHandler`] method for removing previously added locator handlers.
+
+```js
+const locator = page.getByText('This interstitial covers the button');
+await page.addLocatorHandler(locator, async overlay => {
+  await overlay.locator('#close').click();
+}, { times: 3, noWaitAfter: true });
+// Run your tests that can be interrupted by the overlay.
+// ...
+page.removeLocatorHandler(locator);
```
Comment (dgozman) [inline docs/src/release-notes-js.md:47]:
```suggestion
await page.removeLocatorHandler(locator);
```

File: docs/src/release-notes-js.md [hunk 2]
```diff
@@ -6,6 +6,143 @@ toc_max_heading_level: 2
...
+**Command line**
+
+- `--last-failed` CLI option to for running only tests that failed in the previous run.
+
+  First run all tests:
+  ```sh
+  $ npx playwright test
+
+  Running 103 tests using 5 workers
+  1) [chromium] › my-test.spec.ts:8:5 › two ────────────────────────────────────────────────────────
```
Comment (dgozman) [inline docs/src/release-notes-js.md:81]: I'd replace lines 81-117 with `...`

File: docs/src/release-notes-js.md [hunk 3]
```diff
@@ -6,6 +6,143 @@ toc_max_heading_level: 2
...
+      18 | for (let i = 0; i < 100; i++) {
+
+        at /Users/yurys/sandbox/pwtest/tests/my-test.spec.ts:15:13
+
+  2 failed
+    [chromium] › my-test.spec.ts:8:5 › two ─────────────────────────────────────────────────────────
+    [chromium] › my-test.spec.ts:13:5 › three ──────────────────────────────────────────────────────
+  101 passed (30.0s)
+  ```
+  Fix the only failing tests and re
```
Comment (dgozman) [inline docs/src/release-notes-js.md:120]: Remove lines 120-126?

**Review by dgozman — APPROVED** [review]

File: docs/src/release-notes-js.md [hunk 4]
```diff
@@ -6,6 +6,104 @@ toc_max_heading_level: 2
...
+- [`method: LocatorAssertions.toHaveAccessibleName`] checks if the element has the specified accessible name:
+  ```js
+  const locator = page.getByRole('button');
+  await expect(locator).toHaveAccessibleName('Submit');
+  ```
+
+- [`method: LocatorAssertions.toHaveAccessibleDescription`] checks if the element has the specified accessible description:
+  ```js
+  const locator = page.getByRole('button');
+  await expect(locator).toHaveAccessibleName('Upload the photo');
```
Comment (StanislavGrishaev) [inline docs/src/release-notes-js.md:24]: probably `await expect(locator). toHaveAccessibleDescription('Upload the photo');` ?

**Pull request merged**